packages/ui-components/src/components/text-field/text-field.tsx

```diff
@@ -0,0 +1,69 @@
+import { Component, Event, EventEmitter, h, Host, Prop } from '@stencil/core';
+import { EInputFieldType, IErrorMsgs } from './text-field.types';
+
+@Component({
+	tag: 'kv-text-field',
+	styleUrl: 'text-field.scss',
+	shadow: true
+})
+export class KvTextField {
```

Comment:
Do we have to implement the slim textfield at the moment? Please discuss this with the design team.

https://zeroheight.com/1b10805cd/p/41c9b5--text-field/b/87b9f3